src/github.com/couchbaselabs/sync_gateway/rest/doc_api.go

```diff
@@ -75,17 +75,34 @@ func (h *handler) handleGetDoc() error {
 			return base.HTTPErrorf(http.StatusBadRequest, "bad open_revs")
 		}
 
-		err = h.writeMultipart("mixed", func(writer *multipart.Writer) error {
+		if h.requestAccepts("multipart/") && !h.requestAccepts("application/json") {
```

Comment:
this logic is cargo-culted and I can't know if it is correct until I know what the accept header looks like coming from clients that should get multipart.